be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -856,6 +857,18 @@ void SegmentIterator::_evaluate_short_circuit_predicate(uint16_t* vec_sel_rowid_
     for (auto column_predicate : _short_cir_eval_predicate) {
         auto column_id = column_predicate->column_id();
         auto& short_cir_column = _current_return_columns[column_id];
+        auto* col_ptr = short_cir_column.get();
```

Comment:
Please add a todo for code refactor here.

Reply:
done